src/formats/PDB.cpp

```diff
@@ -85,6 +88,9 @@ void PDBFormat::read(Frame& frame) {
         case Record::CONECT:
             read_CONECT(frame, line);
             break;
+        case Record::MODEL:
+            ++step_;
```

Comment:
Also, why are you incrementing the step here instead of after the `end:` label? Does this means that the step will not be incremented in case of file without `MODEL` record?

Reply:
Yeap. `step_` is just to keep track of the `MODEL` records.